web/gds-ui/src/components/LookupForm.js

```diff
@@ -27,7 +29,7 @@ class Lookup extends React.Component {
   }
 
   render() {
-    const detectedType = this.state.inputType !== '' ? `Detected input type: ${this.state.inputType}` : '';
+    const detectedType = this.state.inputType !== '' ? i18n._(t`Detected input type: ${this.state.inputType}`) : '';
```

Comment:
I'm looking through the lingui documentation, is the full ```i18n._(t`hello world`)``` required, or would just 

```
t`hello world`
```

work? The document seems to suggest it would?

Reply:
Yes I think only t macro works too. Should I change all `i18n._(t )` to only t macros?

Reply:
That would certainly make things a lot simpler; would it be an easy find and replace? If so, that would be helpful for future development!

Reply:
Yes it was an easy change. Just made the commit and made sure it didn't mess with any messages extracted by lingui (though there were line number changes to the `po` files).